### PR TITLE
UNIX domain socket support for GSB

### PR DESCRIPTION
**Pull request by Wiezzel**

### CODE REVIEW

File: crates/router/src/lib.rs [hunk 1]
```diff
@@ -9,10 +9,12 @@ use std::fmt::{Debug, Display};
 use std::hash::Hash;
 use std::net::SocketAddr;
 use std::sync::Arc;
-use tokio::net::{TcpListener, TcpStream};
+use tokio::net::{TcpListener, TcpStream, UnixListener, UnixStream};
```
Comment (mfranciszkiewicz) [inline crates/router/src/lib.rs]: Not platform independent

**Review by mfranciszkiewicz — CHANGES_REQUESTED** [review]

**Review by mfranciszkiewicz — APPROVED** [review]

**Review by mfranciszkiewicz — DISMISSED** [review]

**Review by maaktweluit — APPROVED** [review]
LGTM!

**Comment by Wiezzel** [comment]
@mfranciszkiewicz Issue with ya-provider resolved by setting `DEFAULT_GSB_URL` to `unix:/tmp/yagna.sock` on UNIX.

**Review by mfranciszkiewicz — APPROVED** [review]

**Comment by Wiezzel** [comment]
@maaktweluit Bump

**Review by maaktweluit — APPROVED** [review]
Had reviewed this already last thursday :S 

LGTM!

Should this go to `payments/beta-1` or `release/v0.6` to keep in sync with yagna?

**Pull request merged**